probe_src/probe_frontend/src/display.rs

```diff
@@ -0,0 +1,302 @@
+use std::fmt::Display;
+
+use crate::ops;
+use chrono::{DateTime, SecondsFormat};
+
+impl Display for ops::statx_timestamp {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match DateTime::from_timestamp(self.tv_sec, self.tv_nsec) {
+            Some(x) => f.write_str(&x.to_rfc3339_opts(SecondsFormat::Secs, true)),
+            None => f.write_str("[INVALID TIMESTAMP]"),
+        }
+    }
+}
+
+impl Display for ops::timeval {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match DateTime::from_timestamp(self.tv_sec, self.tv_usec as u32 * 1000) {
+            Some(x) => f.write_str(&x.to_rfc3339_opts(SecondsFormat::Secs, true)),
+            None => f.write_str("[INVALID TIMESTAMP]"),
+        }
+    }
+}
+
+impl Display for ops::statx {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f,
```

Comment:
I have half a mind that the `Display` should only have `inode`, `uid`, `gid`, and `mode` for ease of human understanding. All the attributes would still be present in jsonlines. Thoughts?

Reply:
Agreed, one of the things I was going to bring up was if some of the fields of various structs should be omitted.

Reply:
Addressed by https://github.com/charmoniumQ/PROBE/pull/20/commits/6573d3cec29bdfb5f4c9027b142ba260ec59bb6e.